index.html

```diff
@@ -1048,16 +1048,7 @@
     </ul>
    </section>
 
-   <section>
-     <h2>Assisted Reader Technology</h2>
-       <p>TBD</p>
-     <ul class="use-cases">
-       <li>As a user of assistive technology, Alice wants to perceive the full
-    PWP.</li>
-   </ul>
-   </section>
-
-   <section>
+    <section>
```

Comment:
I would prefer (that also holds for the PR #43 to remove sections that do not add anything new... But that is my personal preference only